operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md

```diff
@@ -1,50 +1,31 @@
 # Change Air-Cooled Node BMC Credentials
 
-This procedure will use the System Configuration Service (SCSD) to change all air-cooled Node BMCs in the system to the same global credential.
+This procedure describes how to use the System Administrator Toolkit's `sat bmccreds`
+command to set a global credential for all BMCs on air-cooled nodes.
 
 ### Limitations
 
 All air-cooled and liquid-cooled BMCs share the same global credentials. The air-cooled Slingshot switch controllers (Router BMCs) must have the same credentials as the liquid-cooled Slingshot switch controllers.
 
+The `sat bmccreds` command is not able to target air-cooled nodes directly. It can, however, target nodes by their xname. The following procedure uses xnames to target nodes.
```

Comment:
```suggestion
The `sat bmccreds` command is not able to target air-cooled nodes directly. It can, however, target nodes by their component name (xname). The following procedure uses xnames to target nodes.
```